src/Bridge/Doctrine/MongoDbOdm/ItemDataProvider.php

```diff
@@ -64,19 +63,13 @@ public function supports(string $resourceClass, string $operationName = null, ar
      *
      * @throws RuntimeException
      */
-    public function getItem(string $resourceClass, $id, string $operationName = null, array $context = [])
+    public function getItem(string $resourceClass, $identifiers, ?string $operationName = null, array $context = [])
```

Comment:
Not typehinted to avoid BC? Maybe add a typehint comment for 3.0.

Reply:
Indeed I didn't touch the interfaces yet but I can update them, this needs to target 3.0 though.